src/librustc_mir/const_eval/machine.rs

```diff
@@ -186,6 +190,20 @@ impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for CompileTimeInterpreter {
         false
     }
 
+    #[inline(always)]
+    fn stack(
+        ecx: &'a InterpCx<'mir, 'tcx, Self>,
+    ) -> &'a [Frame<'mir, 'tcx, Self::PointerTag, Self::FrameExtra>] {
+        &ecx.machine.stack
+    }
+
+    #[inline(always)]
+    fn stack_mut(
+        ecx: &'a mut InterpCx<'mir, 'tcx, Self>,
+    ) -> &'a mut Vec<Frame<'mir, 'tcx, Self::PointerTag, Self::FrameExtra>> {
+        &mut ecx.machine.stack
+    }
```

Comment:
These are added in the middle of a group of `enforce_*` hooks, please move them down a bit. (Down one item should be fine for now.)

Reply:
Oh this is the const-eval instance, not the trait def.n, I misread... then ideally make the order consistent with the trait def.n.

Reply:
Moved in both const eval and const prop.